src/app/modules/main/module.nim

```diff
@@ -333,6 +333,12 @@ proc createCommunitySectionItem[T](self: Module[T], communityDetails: CommunityD
     # We will update the model later when we finish loading the accounts
     self.controller.asyncGetRevealedAccountsForAllMembers(communityDetails.id)
 
+    # If there tokens are already in the model, we should keep the existing community tokens, until
+    # getCommunityTokensDetailsAsync will trigger onCommunityTokensDetailsLoaded
+    let existingCommunity = self.view.model().getItemById(communityDetails.id)
```

Comment:
is ti possible that `existingCommunity` in nil here?

Reply:
Yes, good point! Fixed
(it's not nil because getItemById returns by value, so by default it creates an empty community with an empty tokenlist)